chapter3.adoc

```diff
@@ -244,3 +244,30 @@ prior to the `MINVAL.SPA`, and
 subsequent to the `MFENCE.SPA`
 
 `MINVAL.SPA` is only valid in M-mode.
+
+=== Extension to behavior of SFENCE.VMA and HFENCE.VMA
+
+The behavior of the SFENCE.VMA and the HFENCE.VMA instructions are affected when
```

Comment:
HFENCE.VMA -> HFENCE.GVMA and HFENCE.VVMA.

Should also add a note about their Svinval equivalent.

Reply:
To be thorough, we would need another paragraph for the HFENCE instructions similar to the SFENCE V=0 case saying that "the effective VMID is the combination of the SDID and VMID".